analysis/connection.R

```diff
@@ -0,0 +1,45 @@
+# author: Daniel Câmara
+# date: March 3rd, 2022
+
+system2("ssh", c("epigraph@epigraphhub.org -L 5432:localhost:5432", "-NC"), wait = FALSE)
```

Comment:
This command should be issued after loading the `.env`file so that in can use the values specified in there instead of have them hardcoded here. Also, be aware that for opening the SSH connection, the user will be asked for the her/his ssh passphrase. Make sure this interaction is possible with the system2 function.

Reply:
@fccoelho I've been trying to figure a way to solve this for the past several days but unfortunately I'm unable to figure your request. I've went through several packages and forums but could not find how to attend. I'm gonna need some help with this.

Reply:
@dcpcamara, this can be left  simply as an instruction for the user to do it her/himself on the terminal. Don't get hung up on this.

Reply:
Thank you very much for this. However, since I've been trying to solve this, I believe that the new push is making the connection more easily (and having your suggestions also implemented).